### ISSUE TITLE
YoHours : allow use on tactile devices

### ISSUE THREAD

**Issue by GDoukhan** [PanierAvide/panieravide.github.io#38]
Selecting the columns is not possible on tactile devices, too bad.
Even if it would be necessary to also have responsive design to allow a good use on small tactile devices like smartphones

**Comment by PanierAvide** [comment]
Of course, this feature has sense. However, the technology used for showing calendar doesn't allow to do this, so it would need a huge of rework to make this possible.
